snowpack/src/sources/local.ts

```diff
@@ -453,7 +457,7 @@ export default {
           // above have a chance to enter the queue. Prevents a premature exit.
           await new Promise((resolve) => setTimeout(resolve, 5));
```

Comment:
What is this line doing? Should we be waiting for each `this.resolvePackageImport()` above?

Reply:
Tried to document this in the comment above, feel free to edit if it could be more clear:

`resolvePackageImport()` is an async function, so iirc everything inside of it executes separately. But, it can add new jobs to `inProgressBuilds`, and so we don't want to end this run and potentially make `inProgressBuilds` empty, if we're about to add new jobs to it.

 The way Node works, an async function call or new promise gets pushed onto a stack of things to do, and then on the next run of the event loop those get executed. So we really just want to wait for one pass of the event loop, to give `resolvePackageImport()` a chance to add new jobs to the queue.

Reply:
Got it. That makes sense. For me I would have expected `process.nextTick()` or something similar if that was what was happening

Reply:
yea this was me being a bit cautious, now that I've had some time away I'd actually like to see `process.nextTick()` (don't need to tackle in this PR, just sharing)